src/styles/blocks/card.css

```diff
@@ -0,0 +1,93 @@
+.product-card {
+  box-sizing: border-box;
```

Comment:
It's better to move it to the reset file - * {box-sizing: border-box}

Reply:
Agree, add it to * selector, you've used it in more than one place